packages/makecode-browser/src/languageService.ts

```diff
@@ -154,6 +154,10 @@ export class BrowserLanguageService implements LanguageService {
         });
     }
 
+    async buildSimJsInfoAsync(result: CompileResult): Promise<BuiltSimJsInfo> {
+        throw new Error('Not implemented');
```

Comment:
Would it be helpful to include a comment here to the future developer needing to implement this? Some hint about what's required or where to look for a reference?

Reply:
Added a comment.